packages/shared/fastfetchbot_shared/services/scrapers/config.py

```
@@ -1,137 +1,205 @@
+            return val if val else 3000
+        except (ValueError, TypeError):
+            return 3000
+
+
+settings = ScrapersSettings()
+
+# --- Non-settings module-level objects ---
 
 # Templates & Jinja2
 templates_directory = os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates")
 JINJA2_ENV = Environment(
```

Ruff (0.15.7)

[error] By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)